db_sql.go

```diff
@@ -469,89 +580,161 @@ func SqlCreateDatabase(conx *DBConnection) error {
 	}
 }
 
-func MigrateSchema(conx *DBConnection, schema_name string, migrations []string) error {
-	return migrateSchemaInternal(true, conx, schema_name, migrations)
-}
+func RunMigrations(conx *DBConnection) (bool, error) {
+	// Authaus used its own migration tool, but we moved away from that and started using a migration tool by BurntSushi. This requires us to delete old version tracking tables etc.
```

Comment:
This comment like this should include a genuine date. For example,
"Until March 2016, Authaus used it's own migration tool...."